### PR TITLE
Switch versioning system to use autover

### PR DESCRIPTION
**Pull request by jlstevens**
PR to switch geoviews to autover, following the corresponding PR in https://github.com/ioam/holoviews/pull/2426. Embedding version.py until it is available via param.

The only thing I'm not entirely sure about is whether a MANIFEST will need to be added for the ``.version`` file.

### CODE REVIEW

**Comment by jlstevens** [comment]
I have released 1.5.0a1 [here](https://anaconda.org/ioam/geoviews/files) using this PR (and pushed the tag). As far as I can tell, the tests are only failing because of a few flake issues.

**Comment by jlstevens** [comment]
@philippjfr I think this is ready to review/merge when the tests pass and when you confirm that the alpha works as expected.

**Comment by jlstevens** [comment]
I think this PR is now ready to review/merge.

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ __init__.py export-subst
+setup.py export-subst
```
Comment (philippjfr) [inline .gitattributes]: Newlines here and in .gitignore would be nice, but won't let it hold up the PR.

**Comment by philippjfr** [comment]
One minor comment about newlines, I'll let you decide what to do, happy to merge either way.

File: .gitattributes (hunk 1 above)
Comment (jlstevens) [reply .gitattributes]: Addressed in 3efb09f

**Comment by jlstevens** [comment]
Addressed your comment and the tests are green.

**Pull request merged**